hooks/_common.sh

```diff
@@ -468,6 +508,8 @@ function common::terraform_init {
   local exit_code=0
   local init_output
 
+  TF_BINARY=$(common::get_tf_binary)
```

Comment:
For the consistency's sake
```suggestion
  TF_PATH=$(common::get_tf_binary)
```